programs/sbf/tests/programs.rs

```diff
@@ -2242,10 +2242,7 @@ fn test_program_sbf_disguised_as_sbf_loader() {
             mint_keypair,
             ..
         } = create_genesis_config(50);
-        let mut bank = Bank::new_for_tests(&genesis_config);
-        bank.deactivate_feature(
-            &solana_sdk::feature_set::remove_bpf_loader_incorrect_program_id::id(),
-        );
```

Comment:
This doesn't look like it belongs in this PR. Is the removal of this feature related to the sBPF test crash?

Reply:
I can split it into a separate PR. However, I discovered that this doesn't deactivate the feature when I removed `ProgramTest`. That is because it is already to late when the bank is constructed. Features need to be deactivated with the genesis accounts, because that is when the program runtime environment is initialized.

Reply:
I would advise splitting it out, especially if there's more work to be done to disable it.